content/docs/user-guide/assets/pipeline/asset-cache.md

```diff
@@ -23,20 +23,25 @@ There are three parts to the Asset Cache: the **Asset Database**, the **Asset Ca
 
 The Asset Database is only used by Asset Processor. The Asset Database is a development time database containing details about assets, jobs, **Asset Builders**, and scan directories. It contains all the information Asset Processor requires to determine which assets have been processed, and which assets need to be processed.
 
-With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, remembering all the pending process jobs in the job queue.
+With the information in the Asset Database, Asset Processor can be suspended in the middle of a process job and resume when relaunched, rebuilding the queue of jobs to process.
 
 ### Asset Catalog
 
 The Asset Catalog is a runtime catalog of product assets and product dependencies. The Asset Catalog contains a subset of information from the Asset Database; the information about assets required at runtime. **Asset Browser** uses the Asset Catalog to find the runtime product assets on disk that you request for your projects. 
 
+The information in the Asset Catalog XML file in the cache is not the same information that is available in the runtime project. During development, **O3DE Editor** and game launchers use the `assetcatalog.xml` file for tracking assets. In bundled release builds, the only assets available in the catalog are those that are specifically mounted within [asset bundles.](/docs/user-guide/packaging/asset-bundler/) Each bundle contains a Delta Catalog, which is a subset of the Asset Catalog that contains information about the assets within the asset bundle. As asset bundles are mounted, the Delta Catalog XML file is loaded, and the in-memory Asset Catalog is expanded to contain the information from the Delta Catalog.
+
 ### Asset Cache
 
- The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+The Asset Cache contains the Asset Database, the Asset Catalog, and the product assets generated by Asset Builders.
+
+The Asset Cache contains versions of the product asset for the host platform as well as versions for each target platform. For example, when you create a project for Windows and mobile platforms, the cache contains separate versions of the Asset Cache for Windows and each mobile platform. If you are developing the project on Windows and deploying on Windows, the same product assets are used for the host and target platforms.

```

Comment:
nit: The Asset Cache contains versions of the product asset for the host platform*,* as well as versions for each target platform

Reply:
Fixed